### ISSUE TITLE
V1.5rc installation fails while trying to copy documentation

### ISSUE THREAD

**Issue by DarrenTAnims** [opentoonz/opentoonz#3798]
**Describe the bug**
While installing V1.5rc (from https://github.com/opentoonz/opentoonz/releases/tag/v1.5.0rc) to my Windows PC, I received a warning about copying documentation to my local machine. It warns 1 file at a time, then pauses for about 5 seconds before it warns about the next file. It's a slow process. Some of them copied, but not all.

FYI, I renamed my old stuff folder before installing, so it's creating the folders and files from scratch, not overwriting.
I checked my virus software and I've still got "C:\OpenToonz Stuff" as an exclusion directory, so my virus software shouldn't be causing this issue.
And some of the docs were copied! So maybe it's a problem with just some of them?
In the end I cancelled installation. I spent 20 minutes clicking the button, moving from 1 file to the next and gave up in the end.

**Here's the warning:**
**Screenshots**
![image](https://user-images.githubusercontent.com/28575907/111906218-682ca700-8a47-11eb-849d-3745c70695e2.png)

**And here's how it looks, slowly moving 1 file at a time**
![ScreenCapture](https://user-images.githubusercontent.com/28575907/111906459-8e067b80-8a48-11eb-807b-6085e5dcfdf9.gif)

@shun-iwasawa

**Comment by artisteacher** [comment]
I didn't run into this on macOS. Though the installer always tries to overwrite portable installations of OT (ie. Actions artifacts) instead of installing in Applications as expected. If there are not additional installations on the computer, it's not an issue.

**Comment by shun-iwasawa** [comment]
@DarrenTAnims 
Can you please check the hash value of the package?

- open Command prompt
- type `certutil -hashfile C:\path-to-installer\OpenToonzSetup.exe`  (replace _path-to-installer_ by the correct path)
- Check if the output value is the following:
`40f01a3dc3cefaf4e906efc32cffb5f0cfa30fc7`

If not, the installer was not downloaded correctly.
If yes, I'm not sure why... But I'm bit concerned that you are running on Windows8 (is it right?). I could install with no problem on  all of my computers but they are all Win10.

**Comment by DarrenTAnims** [comment]
Thanks for the suggestions @shun-iwasawa.

1. I checked the hash. I got the same value as you.
2. I'm running Windows 10

**Comment by shun-iwasawa** [comment]
Hmm, here are points as many as I can think of...
1. You set "C:\OpenToonz **S**tuff" as exclusion directory in the virus software. (I think it's not the case, but) by any chance the software is case sensitive?
2. Does your user account have admin privileges? If not, can you try installing with it?
3. Can you try installing while temporarily disabling the virus software?

**Comment by manongjohn** [comment]
You said you put the stuff folder in anti-virus exclusion, but did you do that with the installer itself? What anti virus do you use?

Every once in.a while, i get a warning when running an installer, and need to allow it, but I think the warning is coming from Windows Defender, not my anti virus.

**Comment by DarrenTAnims** [comment]
Woohoo!! It's installed. To respond to your kind comments, for completeness:

@Shun. I checked the case of the directory in my anti-virus software. I'm using AVG.
Yes, my account has admin privileges. I'm the only user on this computer.
I'd not tried disabling my AVG. I did that (or so I thought) and when I did, part way through installing, my software warned me that a program was trying to rename a file in the OpenToonz Stuff/doc folder and offered me an allow/disallow button. I clicked allow, then the rename dialog (that I showed at the top of this report) was visible. I clicked Try again and it continued through the rest of the installation to the end! So this seems all it was: my anti-virus software was doing something secretly from me.

Thanks for the suggestion about turning off anti-virus. I'd seen that advice before, but I'd never had a problem installing OpenToonz before so I didn't even think about it. Schoolboy error. Thanks again.

No @manongjohn, I didn't do anything with my anti-virus prog, re the installer. I'm running it from my downloads folder, which is where I run all downloaded installers from, including all previous OpenToonz installers.
